content/2019-06-02-newsletter-18.md

```diff
@@ -1,6 +1,6 @@
 +++
 title = "The Embedded Working Group Newsletter - 18"
-date = 2019-04-05
+date = 2019-06-02
 draft = true
```

Comment:
```suggestion
draft = false
```
Also let's activate this right away.

Reply:
Shouldn't the ecosystem crate numbers be updated first?